samples/Mvc.ActionConstraintSample.Web/Controllers/ItemsController.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc;
+
+namespace ActionConstraintSample.Web.Controllers
```

Comment:
Yes but the other projects generally don't have namespace / directory mismatches.